cirq/contrib/quimb/mps_simulator_test.py

```diff
@@ -324,7 +340,12 @@ def test_supremacy_equal_more_cols():
 def test_tensor_index_names():
     qubits = cirq.LineQubit.range(12)
     qubit_map = {qubit: i for i, qubit in enumerate(qubits)}
-    state = ccq.mps_simulator.MPSState(qubit_map)
+    state = ccq.mps_simulator.MPSState(
+        qubit_map,
+        prng=value.parse_random_state(0),
+        axes=[],
+        log_of_measurement_results={},
```

Comment:
Please add a test confirming that these are passed through to the base ActOnArgs object. The behavior of these fields should already be tested by that class, so this new test doesn't need to be complex.